www/includes/easyparliament/templates/html/search/form.php

```diff
@@ -12,7 +12,7 @@
                     <?php } ?>
                     <p>
                         <ul class="search-result-display-options">
-                        <li><a href="#options" class="search-options-toggle js-toggle-search-options">Advanced search</a></li>
+                        <li><a href="?show_advanced_options=true" class="search-options-toggle js-toggle-search-options">Advanced search</a></li>
```

Comment:
Yes, this won't really work, I'm afraid. What I was thinking was that nothing had to change server side, but that the form was hidden in CSS rather than JS using a `.js` selector (this might need to be set earlier in the HTML but that'd be good to do anyway).

For bonus points, the advanced search form could be in the HTML placed underneath the search results, and then moved with JavaScript to underneath the main form. That way if you have search results, you see them immediately, rather than the form you've just filled in, and the internal #options advanced search still works nicely.